docs/reference/config.md

```diff
@@ -145,6 +145,17 @@ commitAuthor:
 commitMessage: "chore(release): this is a custom release message"
 ```
 
+## DirtyFiles
+```{ .yaml .annotate linenums="1" }
+# Allowing specific case sensitive files to allow using uplift to
+# add unedited but changed files as part of your CI
+
+git:
+    dirtyFiles:
+        - file.txt
+        - path/to/file.txt
+```
+
```

Comment:
Since this is part of the `git` config, can you merge this in the ##git section rather than having its own ##DirtyFile section